Marlin/src/lcd/ultralcd.cpp

```diff
@@ -3110,11 +3112,32 @@ void kill_screen(const char* lcd_msg) {
   #else
     #define _MOVE_XY_ALLOWED true
   #endif
+	
+  void disable_soft_endstop() {
+	enqueue_and_echo_commands_P(PSTR("M211 S0"));
+	IS_SOFT_ENDSTOP_ENABLED = 1;
+  } 
+  
+  void enable_soft_endstop() {
+  	enqueue_and_echo_commands_P(PSTR("M211 S1"));
+	IS_SOFT_ENDSTOP_ENABLED = 0;
+  }
 
   void lcd_move_menu() {
     START_MENU();
     MENU_BACK(MSG_PREPARE);
-
+	
+	#if ENABLED(MIN_SOFTWARE_ENDSTOPS) && ENABLED(MAX_SOFTWARE_ENDSTOPS) && ENABLED(SHOW_SOFT_ENDSTOP_MENU)
+	// Add Software endstop settings
```

Comment:
Shouldn't this be
```
#if (ENABLED(MIN_SOFTWARE_ENDSTOPS) || ENABLED(MAX_SOFTWARE_ENDSTOPS)) && ENABLED(SHOW_SOFT_ENDSTOP_MENU)```

Reply:
My intention was if one of those 3 variable is disabled, this option will be disabled, which means all 3 options must be enabled in order to user this options. Thought it was right to use "&&"(AND) in this kind of situation?

Reply:
Some software endstops will exist when `ENABLED(MIN_SOFTWARE_ENDSTOPS) || ENABLED(MAX_SOFTWARE_ENDSTOPS)`.